jmaqs-base/src/main/java/com/magenic/jmaqs/base/ConcurrentManagerHashMap.java

```diff
@@ -11,6 +11,8 @@
  */
 public class ConcurrentManagerHashMap extends ConcurrentHashMap<String, BaseTestObject> {
 
+  private static final long serialVersionUID = 1;
```

Comment:
comment for what this variable is used for

Reply:
More importantly, why is it there?  The field is `private static final` and it is not used anywhere in the class.

Reply:
https://stackoverflow.com/questions/30716967/is-java-serialversionuid-of-1l-ok-or-does-it-need-to-be-unique

Reply:
https://stackoverflow.com/questions/30962559/why-java-declared-serialversionuid-in-exception-and-throwable-class

Seems like it has value in exceptions

Reply:
Based on the research, the serialVersionUid is for interoperability throughout the exceptions stack.  Certain aspects of exception will not be able to be thrown if the exception is not serializable.  So serialVersionUid is ok in exceptions and serializble inheritance classes, but are out of place in normal classes.